src/applications/yellow-ribbon/components/SearchResultsPage/index.js

```diff
@@ -55,11 +55,6 @@ export const SearchResultsPage = ({
           />
         </button>
 
-        {/* Search Form Header */}
-        <h2 className="vads-u-display--none vads-u-font-size--h3 vads-u-margin-top--1p5 medium-screen:vads-u-display--flex">
```

Comment:
Removing this will impact a11y as now we jump from an `h1` to the `h3` on (what is now) L63. The `<SearchForm>` component uses a `fieldset > legend` for the "Search criteria' label, which I think is correct, so I think we should increase the header level for "Helpful links"

Reply:
Awesome thanks! 

This user story that is in flight currently will tackle those things you mentioned above. 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/9374

Reply:
For this one, I am just removing the dup.